### PR TITLE
Update activity_login.xml

### LINKED ISSUE #1601: Redesign ui of login screen as per new design
Old Screenshot:

New Screenshot:

### PR DESCRIPTION
**Pull request by Prashant830**
I  have create two edit box and modify background color and font in login button at activity_login.xml

Fixes #1601 
before changes
WhatsApp Image 2020-12-31 at 11 05 46 AM

after changes
WhatsApp Image 2020-12-31 at 10 59 15 AM

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt [hunk 1]
```diff
@@ -229,4 +232,3 @@ class RegistrationFragment : BaseFragment(), RegistrationView {
             return RegistrationFragment()
         }
     }
```
Comment (PatelVatsalB21) [inline app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt:234]: @Prashant830 I think you are missing closing curly brace at end that cause gradle build fail. Please run `.\gradlew check  --scan` before commit. Also with this error you might not have successfully build project in your android studio it should have shown you error. Please confirm errors before commiting them. Also there are commits of different error in this pr. Make different branches for different pr. You can find help for git and github on internet.

Comment (Prashant830) [reply app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt:234]: @PatelVatsalB21 you have a code for deleting  specific PR ??

Comment (Prashant830) [reply app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt:234]: In git..

Comment (PatelVatsalB21) [reply app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt:234]: @Prashant830   use `git reset hash` (here you need to place hash in numbers of commit previous to yours ) to go back to last merged commit and that will revert your commit locally then you remove unwanted changes in android studio and commit back then use `git push -f`. I would suggest to go deep on youtube as explaining here would be difficult.

Comment (Prashant830) [reply app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt:234]: @PatelVatsalB21 thanks bro..

Comment (Prashant830) [reply app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt:234]: hello brother you have a LinkedIn account ?? If yes then  give me your account link.. I want to discuss with  you one my idea..

Comment (PatelVatsalB21) [reply app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt:234]: @Prashant830 you can connect via [This](https://www.linkedin.com/in/vatsal-patel-919691193) :)

**Comment by garvit984** [comment]
@Prashant830 This PR does not follow the design that we proposed so we are closing it.

**Comment by Prashant830** [comment]
> @Prashant830 This PR does not follow the design that we proposed so we are closing it.
ok
i got it
I ddidn't know because I made an open source contribution for the first time.

**Pull request closed without merging**